cypress/fixtures/postcodes.json

```diff
@@ -1,4 +1,5 @@
 {
   "default": "B12JP",
+  "testPostcode": "L1 0BG",
```

Comment:
We agree that we don't need this and need to check if we're not seeing courses why is that because on QA they are visible.